src/main/java/org/myrobotlab/service/WebGui.java

```diff
@@ -1320,4 +1322,24 @@ public static void main(String[] args) {
       log.error("main threw", e);
     }
   }
+
+  @Override
+  public void onCreated(String name) {

```

Comment:
shouldn't there be some implementation here?  if not, why is WebGui implementing the ServiceLifeCycleListener?

Reply:
webgui has to keep the ui in sync with any new service, it does so by subscribing to lifecycle events..
it only needs the onStarted

if you remember synching the availability of services for attaching has gone through a couple iterations ...
I'm not positive that WebGui does not need the onStarted events but I didn't want to keep adding changes to this pr